### PR TITLE
fix : toc link overflow

### PR DESCRIPTION
**Pull request by heygsc**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

image
Exceeding parts, text cannot be displayed

### CODE REVIEW

File: docs/.vitepress/vitepress/styles/content/table-of-content.scss [hunk 1]
```diff
@@ -44,19 +44,22 @@
         margin-top: 10px;
         font-size: 11px;
         color: var(--text-color-lighter);
-        text-overflow: ellipsis;
-        overflow: hidden;
-        white-space: nowrap;
         color: inherit;
```
Comment (btea) [inline docs/.vitepress/vitepress/styles/content/table-of-content.scss:47]:
```suggestion
        color: inherit;
        list-style: none;
```

**Comment by btea** [comment]
The default style of the `li` tag needs to be adjusted.
image

**Comment by btea** [comment]
@heygsc You don't seem to make any modifications.

File: docs/.vitepress/vitepress/styles/content/table-of-content.scss (hunk 1 above)
Comment (heygsc) [reply docs/.vitepress/vitepress/styles/content/table-of-content.scss:47]: > 

Maybe it's my first time trying to commit to a review, but now it seems like I've changed it.

**Review by btea — APPROVED** [review]

**Comment by heygsc** [comment]
@btea   What do I need to do more for this pr? Or just waiting for merge?

**Comment by btea** [comment]
You don't need to do anything now, just wait for another member to review it.

**Review by tolking — APPROVED** [review]

**Pull request merged**